gnowledge/gstudio
# Templates for error pages added

**Templates for error pages added:**
- Template for `400` and `500` errors has been added.
- Created base template `error_base.html` for error templates.
  - There was need to create this simple template rather than using `base.html`.
  - `base.html` has become so dependent on `ndf_tags`, without proper arguments it doesn't render.
  - And it's recommended to extend simple template or to create simple template/s without data variables to avoid further errors.

**Other Changes:**
- Footer part has been removed from `base.html` and created new template `footer.html`.

--

**TESTING**
Need some changes in `settings.py` and the way we run the project to test locally:
- update/add following in `settings.py` file and save it:
  - `DEBUG = False`
  - `ALLOWED_HOSTS = ["127.0.0.1", "localhost"]`
- Running the project ([read more about this](https://docs.djangoproject.com/en/1.6/ref/contrib/staticfiles/#django-admin-option---insecure)):
  - `python manage.py runserver --insecure`
- Try to get 404 and 500 error by changing urls.
- Also try to load normal pages. Check if they gets loaded with proper static resources.
